recipe_sha: 075530121157793ebc0bb40fef37ca076593c28fb131f9ad82ce44f43271c471  # The hash of the recipe's meta.yaml at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |2-
    116 |   template<typename _Tp> const _Tp* end(const valarray<_Tp>&) noexcept;
        |                                     ^~~
  /opt/conda/conda-bld/dechat_1734218819382/work/gatb-core/gatb-core/thirdparty/kff-cpp-api/merge.cpp:91:91: error: request for member 'find' in 'footer_values', which is of non-class type 'int'
     91 |                                                                         if (footer_values.find(tuple.first) == footer_values.end())
        |                                                                                           ^~~~
  /opt/conda/conda-bld/dechat_1734218819382/work/gatb-core/gatb-core/thirdparty/kff-cpp-api/merge.cpp:91:126: error: request for member 'end' in 'footer_values', which is of non-class type 'int'
     91 |                                                                         if (footer_values.find(tuple.first) == footer_values.end())
        |                                                                                                                              ^~~
  /opt/conda/conda-bld/dechat_1734218819382/work/gatb-core/gatb-core/thirdparty/kff-cpp-api/merge.cpp:100:69: error: 'begin' was not declared in this scope; did you mean 'std::begin'?
    100 |                                                 for (auto & p : sgv.vars)
        |                                                                     ^~~~
        |                                                                     std::begin
  /opt/conda/conda-bld/dechat_1734218819382/_build_env/lib/gcc/x86_64-conda-linux-gnu/13.3.0/include/c/bits/range_access.h:114:37: note: 'std::begin' declared here
    114 |   template<typename _Tp> const _Tp* begin(const valarray<_Tp>&) noexcept;
        |                                     ^~~~~
  /opt/conda/conda-bld/dechat_1734218819382/work/gatb-core/gatb-core/thirdparty/kff-cpp-api/merge.cpp:100:69: error: 'end' was not declared in this scope; did you mean 'std::end'?
    100 |                                                 for (auto & p : sgv.vars)
        |                                                                     ^~~~
        |                                                                     std::end
  /opt/conda/conda-bld/dechat_1734218819382/_build_env/lib/gcc/x86_64-conda-linux-gnu/13.3.0/include/c/bits/range_access.h:116:37: note: 'std::end' declared here
    116 |   template<typename _Tp> const _Tp* end(const valarray<_Tp>&) noexcept;
        |                                     ^~~
  /opt/conda/conda-bld/dechat_1734218819382/work/gatb-core/gatb-core/thirdparty/kff-cpp-api/merge.cpp:110:57: error: 'begin' was not declared in this scope; did you mean 'std::begin'?
    110 |                                         for (auto & p : variables) {
        |                                                         ^~~~~~~~~
        |                                                         std::begin
  /opt/conda/conda-bld/dechat_1734218819382/_build_env/lib/gcc/x86_64-conda-linux-gnu/13.3.0/include/c/bits/range_access.h:114:37: note: 'std::begin' declared here
    114 |   template<typename _Tp> const _Tp* begin(const valarray<_Tp>&) noexcept;
        |                                     ^~~~~
  /opt/conda/conda-bld/dechat_1734218819382/work/gatb-core/gatb-core/thirdparty/kff-cpp-api/merge.cpp:110:57: error: 'end' was not declared in this scope; did you mean 'std::end'?
    110 |                                         for (auto & p : variables) {
        |                                                         ^~~~~~~~~
        |                                                         std::end
  /opt/conda/conda-bld/dechat_1734218819382/_build_env/lib/gcc/x86_64-conda-linux-gnu/13.3.0/include/c/bits/range_access.h:116:37: note: 'std::end' declared here
    116 |   template<typename _Tp> const _Tp* end(const valarray<_Tp>&) noexcept;
        |                                     ^~~
  /opt/conda/conda-bld/dechat_1734218819382/work/gatb-core/gatb-core/thirdparty/kff-cpp-api/merge.cpp:111:73: error: request for member 'find' in 'outfile.Kff_file::global_vars', which is of non-class type 'int'
    111 |                                                 if (outfile.global_vars.find(p.first) == outfile.global_vars.end()
        |                                                                         ^~~~
  /opt/conda/conda-bld/dechat_1734218819382/work/gatb-core/gatb-core/thirdparty/kff-cpp-api/merge.cpp:111:110: error: request for member 'end' in 'outfile.Kff_file::global_vars', which is of non-class type 'int'
    111 |                                                 if (outfile.global_vars.find(p.first) == outfile.global_vars.end()
        |                                                                                                              ^~~
  /opt/conda/conda-bld/dechat_1734218819382/work/gatb-core/gatb-core/thirdparty/kff-cpp-api/merge.cpp:122:65: error: 'begin' was not declared in this scope; did you mean 'std::begin'?
    122 |                                                 for (auto & p : variables) {
        |                                                                 ^~~~~~~~~
        |                                                                 std::begin
  /opt/conda/conda-bld/dechat_1734218819382/_build_env/lib/gcc/x86_64-conda-linux-gnu/13.3.0/include/c/bits/range_access.h:114:37: note: 'std::begin' declared here
    114 |   template<typename _Tp> const _Tp* begin(const valarray<_Tp>&) noexcept;
        |                                     ^~~~~
  /opt/conda/conda-bld/dechat_1734218819382/work/gatb-core/gatb-core/thirdparty/kff-cpp-api/merge.cpp:122:65: error: 'end' was not declared in this scope; did you mean 'std::end'?
    122 |                                                 for (auto & p : variables) {
        |                                                                 ^~~~~~~~~
        |                                                                 std::end
  /opt/conda/conda-bld/dechat_1734218819382/_build_env/lib/gcc/x86_64-conda-linux-gnu/13.3.0/include/c/bits/range_access.h:116:37: note: 'std::end' declared here
    116 |   template<typename _Tp> const _Tp* end(const valarray<_Tp>&) noexcept;
        |                                     ^~~
  /opt/conda/conda-bld/dechat_1734218819382/work/gatb-core/gatb-core/thirdparty/kff-cpp-api/merge.cpp:144:54: error: 'buffer' was not declared in this scope; did you mean 'setbuffer'?
    144 |                                         infile->read(buffer, size_to_copy);
        |                                                      ^~~~~~
        |                                                      setbuffer
  /opt/conda/conda-bld/dechat_1734218819382/work/gatb-core/gatb-core/thirdparty/kff-cpp-api/merge.cpp:201:43: error: 'begin' was not declared in this scope; did you mean 'std::begin'?
    201 |                 for (const auto & tuple : footer_values) {
        |                                           ^~~~~~~~~~~~~
        |                                           std::begin
  /opt/conda/conda-bld/dechat_1734218819382/_build_env/lib/gcc/x86_64-conda-linux-gnu/13.3.0/include/c/bits/range_access.h:114:37: note: 'std::begin' declared here
    114 |   template<typename _Tp> const _Tp* begin(const valarray<_Tp>&) noexcept;
        |                                     ^~~~~
  /opt/conda/conda-bld/dechat_1734218819382/work/gatb-core/gatb-core/thirdparty/kff-cpp-api/merge.cpp:201:43: error: 'end' was not declared in this scope; did you mean 'std::end'?
    201 |                 for (const auto & tuple : footer_values) {
        |                                           ^~~~~~~~~~~~~
        |                                           std::end
  /opt/conda/conda-bld/dechat_1734218819382/_build_env/lib/gcc/x86_64-conda-linux-gnu/13.3.0/include/c/bits/range_access.h:116:37: note: 'std::end' declared here
    116 |   template<typename _Tp> const _Tp* end(const valarray<_Tp>&) noexcept;
        |                                     ^~~
  make[2]: *** [ext/gatb-core/src/CMakeFiles/gatbcore-static.dir/build.make:1801: ext/gatb-core/src/CMakeFiles/gatbcore-static.dir/__/thirdparty/kff-cpp-api/merge.cpp.o] Error 1
  make[1]: *** [CMakeFiles/Makefile2:350: ext/gatb-core/src/CMakeFiles/gatbcore-static.dir/all] Error 2
  make: *** [Makefile:156: all] Error 2
  Traceback (most recent call last):
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2558, in build
      utils.check_call_env(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/utils.py", line 404, in check_call_env
      return _func_defaulting_env_to_os_environ("call", *popenargs, **kwargs)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/utils.py", line 380, in _func_defaulting_env_to_os_environ
      raise subprocess.CalledProcessError(proc.returncode, _args)
  subprocess.CalledProcessError: Command '['/bin/bash', '-o', 'errexit', '/opt/conda/conda-bld/dechat_1734218819382/work/conda_build.sh']' returned non-zero exit status 2.

  The above exception was the direct cause of the following exception:

  Traceback (most recent call last):
    File "/opt/conda/bin/conda-build", line 11, in <module>
      sys.exit(execute())
    File "/opt/conda/lib/python3.10/site-packages/conda_build/cli/main_build.py", line 589, in execute
      api.build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 209, in build
      return build_tree(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3712, in build_tree
      packages_from_this = build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2566, in build
      raise BuildScriptException(str(exc), caused_by=exc) from exc
  conda_build.exceptions.BuildScriptException: Command '['/bin/bash', '-o', 'errexit', '/opt/conda/conda-bld/dechat_1734218819382/work/conda_build.sh']' returned non-zero exit status 2.
# Last 100 lines of the build log.
